src/api/example/controllers/example-find-all-controller.js

```diff
@@ -9,3 +19,8 @@ const exampleFindAllController = {
 }
 
 export { exampleFindAllController }
+
+/**
+ * @import { ServerRoute, Request, ResponseToolkit } from '@hapi/hapi'
+ * @import { MongoDBPlugin } from '~/src/helpers/mongodb.js'
```

Comment:
is this working? with using babel module resolutions in the import? Interesting

Reply:
the lint/typecheck seems happy but intelij has mixed results. Oddly ServerRoute & Request work, while ResponseToolkit is flagged as unknown when used from the @import